app/data_models/answer_store.py

```diff
@@ -137,15 +137,15 @@ def remove_all_answers_for_list_item_id(self, list_item_id: str):
     def serialize(self):
         return list(self.answer_map.values())
 
-    def get_escaped_answer(self, answer_id, list_item_id):
+    def get_escaped_answer_value(self, answer_id, list_item_id):
```

Comment:
I think `list_item_id` should be optional (default to `None`), to be consistent with the `get_answer` method.

Reply:
Used ` list_item_id=None` here, I hope that's what you wanted, thanks!